portal/portal-api/api/src/java/org/sakaiproject/portal/beans/bullhornhandlers/AnnouncementsBullhornHandler.java

```diff
@@ -87,13 +87,16 @@
         String siteId = pathParts[3];
 
         SecurityAdvisor sa = unlock(new String[] {AnnouncementService.SECURE_ANNC_READ, AnnouncementService.SECURE_ANNC_READ_DRAFT});
+        AnnouncementMessage message = null;
         try {
-            AnnouncementMessage message
-                = (AnnouncementMessage) announcementService.getMessage(
-                                                entityManager.newReference(ref));
-
-            // If the announcement has just been hidden, remove any existing alerts for it
-            if (e.getEvent().equals(AnnouncementService.EVENT_ANNC_UPDATE_AVAILABILITY) && message.getHeader().getDraft()) {
+            message = (AnnouncementMessage) announcementService.getMessage(entityManager.newReference(ref));
```

Comment:
I'm not changing the previous existing behaviour for that, it would be caught by the last catch. Let me know if you think it's necessary to explicitly check it.